util/constants.py

```diff
@@ -10,3 +10,22 @@
 
 # filled in by oceanutil.py
 CONTRACTS: dict = {}  # [chainID][contract_label] : contract_object
+
+WRAPPED_TOKEN_ADDRS = {
+    1: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    3: "0xc778417e063141139fce010982780140aa0cd5ab",
+    4: "0xc778417E063141139Fce010982780140Aa0cD5Ab",
+    137: "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
+    56: "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
+    # TODO Add more
+}
+
+
+WRAPPED_TOKEN_SYMBOLS = {
+    WRAPPED_TOKEN_ADDRS[1]: "ETH",
+    WRAPPED_TOKEN_ADDRS[3]: "ETH",
+    WRAPPED_TOKEN_ADDRS[4]: "ETH",
+    WRAPPED_TOKEN_ADDRS[137]: "MATIC",
+    WRAPPED_TOKEN_ADDRS[56]: "BNB",
+    # TODO Add more
+}
```

Comment:
These might be better placed in util/networkutil.py.

Also, rather than mapping a chainid int to the value, perhaps make it chain name to the value. Less error-prone.